build.sbt

```diff
@@ -48,7 +41,8 @@ lazy val microservice = Project(appName, file("."))
       "-feature",
       "-unchecked",
       "-language:implicitConversions",
-      "-P:silencer:pathFilters=views;routes;TestStorage"
+      "-P:silencer:pathFilters=routes;TestStorage",
+      "-P:silencer:globalFilters=Unused import"
```

Comment:
"-P:silencer:pathFilters=routes;TestStorage"

please replace this with below line :"-P:silencer:lineContentFilters=^\\w"

Reply:
at-least we need to avoid using "views" in this entry

https://confluence.tools.tax.service.gov.uk/display/DDCWLS/How+to+suppresses+unwanted+warnings+from+Twirl+template

Reply:
Doing that will introduce one new warning:
@import views.html.helpers.commonHelper
I think 'views' still work. Not all suggestions on Confluence are working 100%